media-fonts/nerd-fonts/nerd-fonts-2.1.0.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 2019-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit font
+
+DESCRIPTION="Icon patched versions of popular fonts"
+HOMEPAGE="https://nerdfonts.com/"
+
+KEYWORDS="~amd64 ~arm64 ~x86"
```

Comment:
I know these are fonts, but I can't commit with `~arm64` around as I'm not part of that arch team. Let's follow the procedure.